npu.user.js

```diff
@@ -1106,6 +1343,41 @@ var npu = {
 			var result = (/ctrl=([a-zA-Z0-9_]+)/g).exec(window.location.href);
 			return result ? result[1] : null;
 		},
+
+		getLanguage: function() {
+			// QUESTION: There should be a more reliable way to do this...

```

Comment:
It's better not to do it at all :smile:  I believe this method can be removed; see my comment above.